src/components/header/Header.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import * as S from './Header.styles.js';
+
+const Header = () => {
+  const [open, setOpen] = useState(true);
+
+  const openBar = () => {
+    setOpen(!open);
+  };
+
+  return (
+    <S.HeaderWrapper>
+      <S.Header>
+        <S.Wrapper>
+          <Link to="/" id="logo" onClick={() => openBar(open)}>
+            Weather System
+          </Link>
+          <span onClick={() => openBar(open)}>{open ? '+' : '-'}</span>
+        </S.Wrapper>
+        <S.HeaderRight show={open}>
```

Comment:
please think about more specific name for `HeaderRight`

Reply:
Ok, thanks. Fixed to NavLinks.